src/pages/index.js

```diff
@@ -81,36 +196,80 @@ const editFormValidator = new FormValidator(
   formValidationConfig,
   profileEditForm
 );
-editFormValidator.enableValidation();
-
-export { previewImage, previewFooter, previewImageModal };
 
-//Section
+const avatarEditModal = document.querySelector("#edit-avatar-modal");
+const avatarEditForm = avatarEditModal.querySelector(".modal__form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms